rke/k8s_rke_system_images.go

```diff
@@ -4696,6 +4696,86 @@ func loadK8sRKESystemImages() map[string]v3.RKESystemImages {
 			WindowsPodInfraContainer:  "rancher/kubelet-pause:v0.1.6",
 			Nodelocal:                 "rancher/mirrored-k8s-dns-node-cache:1.15.7",
 		},
+		// Enabled in v2.5.8
+		// Reminder: This template contains Nodelocal image which isn't in templates before v2.3.6
+		// Reminder: This template should remain an exact copy of the template for v2.4.x (No ACI images)
+		"v1.18.18-rancher1-1": {
+			Etcd:                      "rancher/coreos-etcd:v3.4.15-rancher1",
+			Kubernetes:                "rancher/hyperkube:v1.18.18-rancher1",
+			Alpine:                    "rancher/rke-tools:v0.1.74",
+			NginxProxy:                "rancher/rke-tools:v0.1.74",
+			CertDownloader:            "rancher/rke-tools:v0.1.74",
+			KubernetesServicesSidecar: "rancher/rke-tools:v0.1.74",
+			KubeDNS:                   "gcr.io/google_containers/k8s-dns-kube-dns:1.15.2",
+			DNSmasq:                   "gcr.io/google_containers/k8s-dns-dnsmasq-nanny:1.15.2",
+			KubeDNSSidecar:            "gcr.io/google_containers/k8s-dns-sidecar:1.15.2",
+			KubeDNSAutoscaler:         "rancher/cluster-proportional-autoscaler:1.7.1",
+			Flannel:                   "quay.io/coreos/flannel:v0.12.0",
```

Comment:
all images in rancher1-1 need to call image.Mirror since they're not prefixed with rancher/mirrored-. similar to https://github.com/rancher/kontainer-driver-metadata/pull/535/files#diff-946512550baddbb48e0b41424179a72358b7d614165c87b1eb0bf0b09cd617d2R4349

Reply:
Completely missed that, thanks